### PR TITLE
Create Vodafone.pt.xml

### PR DESCRIPTION
**Pull request by ekaris**

### CODE REVIEW

File: src/chrome/content/rules/Vodafone.pt.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	<target host="onenetuk.vodafone.pt" />
+	<target host="negocios.vodafone.pt" />
+	<target host="onenet.vodafone.pt" />
+	<target host="conferencing.vodafone.pt" />
+	<target host="forum.vodafone.pt" />
+	<target host="tvnetvoz.vodafone.pt" />
+	<target host="securenet.vodafone.pt" />
+	<target host="www.facturanahora.vodafone.pt" />
+	<target host="rbtweb.vodafone.pt" />
+	<target host="mail.vodafone.pt" />
```
Comment (J0WI) [inline src/chrome/content/rules/Vodafone.pt.xml]: servs an invalid cert

File: src/chrome/content/rules/Vodafone.pt.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	<target host="sms.vodafone.pt" />
+	<target host="www.agente.vodafone.pt" />
+	<target host="webfecs.vodafone.pt" />
+	<target host="webgen.vodafone.pt" />
+	<target host="webvodafonemail.vodafone.pt" />
+	<target host="admin.net.vodafone.pt" />
```
Comment (J0WI) [inline src/chrome/content/rules/Vodafone.pt.xml]: host timed out

Comment (ekaris) [inline src/chrome/content/rules/Vodafone.pt.xml]: works for me

File: src/chrome/content/rules/Vodafone.pt.xml (hunk 1 above)
Comment (ekaris) [inline src/chrome/content/rules/Vodafone.pt.xml]: Should I remove it or just leave a comment?

File: src/chrome/content/rules/Vodafone.pt.xml (hunk 2 above)
Comment (J0WI) [inline src/chrome/content/rules/Vodafone.pt.xml]: confirm

File: src/chrome/content/rules/Vodafone.pt.xml (hunk 1 above)
Comment (J0WI) [inline src/chrome/content/rules/Vodafone.pt.xml]: Please remove it and leave a comment.

File: src/chrome/content/rules/Vodafone.pt.xml (hunk 2 above)
Comment (ekaris) [inline src/chrome/content/rules/Vodafone.pt.xml]: curl https://admin.net.vodafone.pt
<html>
<head>
<title>Vodafone</title>
</head>
<body>
<meta http-equiv="Refresh" content="0;url=https://admin.net.vodafone.pt/ring/#login">
</body>
</html>

**Comment by ekaris** [comment]
All done

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**